libevmasm/Assembly.cpp

```diff
@@ -74,6 +74,123 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 	}
 }
 
+void Assembly::addAssemblyItemsFromJSON(Json::Value const& _code)
+{
+	solAssert(m_items.empty(), "");
+	solAssert(_code.isArray(), "");
+	for (auto const& jsonItem: _code)
```

Comment:
Is `Json::Value` iterable? If yes, this can be replaced with
```
m_items.insert(m_items.begin(), _code.begin(), code.end());
```

Reply:
Note that I'm not just copying the elements here. For each element I'm calling `createAssemblyItemFromJSON` that will be added to `m_items`. But yeah, maybe there is a nicer syntax for that :D

Reply:
Hmm.. you think its nicer like that?

Reply:
Not really

Reply:
Executive decision ;-): fine as is ;-).